beginner-guides.md

```diff
@@ -0,0 +1,65 @@
+
+This is a step-by-step guide that may help guide you on where to start if you want to contribute to any of our projects listed in <https://github.com/survival>, but should also be helpful if you wanted to contribute to any open source project that lives on GitHub.
+
+
+## What you will need
+
+* You need to know how to use git, either through a GUI applicaton or through the terminal. CodeSchool has [a good tutorial on git](https://try.github.io/levels/1/challenges/1).
+
+* Since most of our projects are in [GitHub](https://github.com), you will need to have an account there. Jessica Lord has [the best tutorial on Github](https://github.com/jlord/git-it-electron).
+
+* It helps to know how to use the terminal as well, since most of our projects need you to run commads, for example, to run the tests. Codebar has a starter tutorial on [the command line](http://tutorials.codebar.io/command-line/introduction/tutorial.html).
+
+
+## Where to start
+
+* Search for issues in any of our repos with the label `good-first-issue`.
+
+* When you find a project you want to contribute to, fork the repository by clicking the "Fork" button on the top right of the project's home page.
+
+![alt text][forkit]
+
+* This will create a copy of our project in your GitHub account. This is because, in order to push to a repository, you need to have writing permissions, and only the owners of a repo have them. But even if you don't have writing permissions to our project, you will have writing permissions to your copy of the project, and you can push code there and then send it to us (will tell you how to do this below).
+
+* Now that you have a copy of our project in your GitHub account, you can clone it to your computer. If you are using the terminal, type `git clone PROJECT_URL_HERE`. You can find the project's URL to clone in your copy of the project, clicking the big green "clone or donwload" button to the right:
+
+![alt text][clone-url]
+
+* The final step is to add our project's url as a remote so that you can fetch changes from it often and keep your copy updated. The URL can be found in the same place where you found your copy's URL. If you are using the terminal, type `git remote add upstream PROJECT_URL_HERE`.
+
+
+## Set up
+
+* If the project is a Ruby project, check that your local Ruby version is the same as in the Gemfile, if not, change it locally. You may need to [install it if you don't have it](https://cbednarski.com/articles/installing-ruby/). How to do this depends on the ruby version manager that you use. Search for its documentation.
+
+* Install `bundler` if you haven't installed it for that version of Ruby (`gem install bundler`)
+
+* Follow the instructions in the README to set the project up, run the tests and the app, and check that everything works before starting your work. If you have any problems contact us: [webmaster@survivalinternational.org](mailto:webmaster@survivalinternational.org)
+
```

Comment:
We should probably create an alias like volunteers@ or contribution@ otherwise it will become hard to filter those emails.

Reply:
I think something like  `open-source@survivalinternational.org` is generic enough to include all code contributions from beginners or experienced people, and not be confused with contributions or volunteers to the organisation that are not code related.

Reply:
I've responded to the issue you created regarding this email.